Container/CSS/list.css

```diff
@@ -0,0 +1,27 @@
+@import url(/Container/CSS/style.css);
+
+body{
+    display: flex;
+    gap: 1rem;
+    flex-direction: column;
+}
+.checkBox-list, .radioBtn-list{
+    display: flex;
+    gap: 1rem;
+    flex-direction: column;
+}
+.checkBox-list li, .radioBtn-list li{
+    font-size: 1.3rem;
+}
+.checkbox{
+    margin-right: 1rem;
+}
+.radio{
+    margin-right: 1rem;
+}
+
+input.checkbox,.radio{
```

Comment:
follow principle of multiple of 4px (0.25rem) or 8px(0.5 rem ) , beacuse its good practice adv ised yesterday

Reply:
will do